src/styles/card.scss

```
@@ -0,0 +1,104 @@
+.product {
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid $border-white;
+  background: $white;
+  padding: 32px 16px 16px;
+
+  &__image {
+    background: url("../images/imac.jpeg") no-repeat;
+    background-size: 160px 134px;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-right: 64px;
+    color: $secondary;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-top: 4px;
+  }
+
+  &__rating {
+    padding-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &_reviews {
+      color: $main-accent;
+      text-align: right;
+      font-family: Roboto, sans-serif;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    padding-top: 23px;
+    display: flex;
+    justify-content: space-between;
+
+
+    &_title {
+      color: $secondary;
+      font-family: Roboto, sans-serif;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 400;
+    }
+
+    &_price {
+      color: $main-accent;
+      text-align: right;
+      font-family: Roboto, sans-serif;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $white;
+
+    box-sizing: border-box;
+    margin: 16px auto 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
```

Need to fix it everywhere
```suggestion
    width: 100%;
```

---

Not fixed